layout/js/animate.js

```diff
@@ -22,10 +22,10 @@ var fetchSvg = $.ajax({
 });
 
 var fetchPrcpColors = $.get("js/precip-colors.json").done(function(data) {
-  prcpColors = data;
+  prcpColors = JSON.parse(data);
```

Comment:
I would recommend changing the fetch as follows (line 24): ```fecthPrcpColors = $.ajax({url:'js/precip-color.json', dataType: 'json'}). Then you shouldn't need the parse.

Reply:
Good call! Thanks.